src/components/Hero/HubHero/index.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react"
+import {
+  Box,
+  Heading,
+  HStack,
+  Stack,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import GatsbyImage from "../../GatsbyImage"
+import { CallToAction } from "../CallToAction"
+import { CommonHeroProps } from "../utils"
+
+export interface HubHeroProps extends CommonHeroProps {
+  title: string
+}
+
+const HubHero = (props: HubHeroProps) => {
+  const { heroImgSrc, title, header, description, buttons } = props
+
+  if (buttons && buttons.length > 2) {
+    throw Error(
+      "Can not have more than two call-to-action buttons in this hero component."
+    )
+  }
+
+  const largeContentBg = useColorModeValue(
+    "rgba(255, 255, 255, 0.80)",
+    "rgba(34, 34, 34, 0.80)"
+  )
+
+  return (
+    <Box position="relative">
+      <GatsbyImage
+        image={heroImgSrc}
+        alt=""
+        w="full"
+        height={{
+          base: "192px",
+          md: "256px",
+          lg: "320px",
+          xl: "576px",
+          "2xl": "672px",
+        }}
+        loading="eager"
+      />
+      <Stack
+        spacing={{ base: "3", md: "4" }}
+        p={{ base: "4", lg: "8" }}
+        textAlign={{ base: "center", xl: "start" }}
+        borderRadius={{ xl: "base" }}
+        bg={{ xl: largeContentBg }}
+        position={{ xl: "absolute" }}
+        insetStart={{ xl: "8" }}
+        maxW={{ xl: "sm" }}
+        top={{ xl: "50%" }}
+        transform={{ xl: "translateY(-50%)" }}
+        backdropFilter={{ xl: "auto" }}
+        backdropBlur={{ xl: "base" }}
+      >
+        <Heading
```

Comment:
@pettinarip I brought this over from current usage in the hub pages. Should this really be a heading? And more importantly, should this be an `h1` versus the other heading below?

![image](https://github.com/ethereum/ethereum-org-website/assets/65234762/0ef2b9a8-fb89-4a54-ad4c-ed9087f701f3)
Here, `Learn Hub` is the h1, whereas the more prominent `Learn About Ethereum` is an h2.

I see this as semantics in this area of the page, though there are two things to point out as far as a screen reader is concerned:
1. `Learn Hub` is already used in the `title` tag for this page, so when the user enters this page, the screen reader will already declare this before the user begins navigating. So from that standpoint, giving the `Learn Hub` heading an `h1` is redundant. Having it as the `Text` component using `span` semantically would be better: along with the redundancy it is not labeling a section (including what I describe in # 2) and the styling lends to this usage.
2.  `Learn About Ethereum` is really the title that describes the page. And `h1` as the heading that should only be used once per page fits the bill here.

Reply:
Thanks for sharing these observations and thoughts.

As far as I remember, the reasons behind these semantics were SEO and the way Algolia indexes the content.

A clear example is in the [layer 2 page](https://ethereum.org/en/layer-2/):
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/92f34391-6a7b-40c1-8dd7-57f6761aa865)

The idea is to have the "formal" and more concrete heading title at the top of the ranking. The current h1, in this case, "Layer 2". That one will appear in the search engines, and that title should not require context to be understood.

The second title, in this case, "Ethereum for everyone" is the more "friendly" subtitle that needs the context of the page to be understood but we don't want that to be the first thing to pop up in search results. That's why it has an h2.

Hope that clarifies a bit the logic here.

cc @wackerow I'm sure you could confirm or correct me if I'm wrong.

======

Then, in terms of SR, you are right. Wondering if we could somehow fix that 🤔 without losing the previously mentioned things.

Not the most ideal solution but I guess that we could have a `sr-only` class for these cases, no?

Reply:
Oh! For my awareness at least I will take a look at SEO conventions related to this.

Ok, I understand now with the h2 usage!

> Not the most ideal solution but I guess that we could have a sr-only class for these cases, no?

That would mean the text is not visible on the page and the screen reader will read it out. Not the case here. There would be the screen reader ignoring the text, but you don't want it to ignore a heading.

I'll revisit this along with the SEO. I will take no action on the code for now because of the explanation with the large subtitle.

Reply:
Ah right! yea, `sr-only` is not going to work in this case.